src/main/resources/application.yml

```diff
@@ -69,9 +69,6 @@ observe:
     judicialResults-isConvictedResult:
       path: "hearing.prosecutionCases[*].defendants[*].offences[*].judicialResults[*].isConvictedResult"
       printValue: true
-    judicialResults-type-description:
```

Comment:
I've removed this because it's quite wordy and has things like email addresses in it which we shouldn't be logging.